_posts/2022-08-14-aug-update.markdown

```diff
@@ -0,0 +1,171 @@
+<br>
+
+We've also been hosted at **FOSDEM 2022**, where Alexander went over the basics of IPC and the general architecture of the system.
+<div style="width: 560px; margin: 0 auto;"><iframe width="560" height="315" src="https://video.fosdem.org/2022/D.microkernel/agrinten.mp4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+<br>
+
+## Major Updates
+
+Major updates since our last post include a 64-bit ARM port, the start of the RISC-V port, support for the [Rust](https://www.rust-lang.org/) programming language in user space, and support for the [xbps package manager](https://github.com/void-linux/xbps).
+
+Another important addition is our
+[handbook](https://docs.managarm.org/handbook) that describes parts of the
+system in detail (targetting both users and developers). The handbook is still
+quite incomplete but regular updates can be expected in the future.
+
+### AArch64 Port
+
+Since July 2020, we have been working on the 64-bit ARM
+(= [AArch64](https://en.wikipedia.org/wiki/AArch64)) port of Managarm. In its
+current form, the port is able to boot into Weston and kmscon and run some
+command-line programs in QEMU. However, a large part of our software repertoire
+is still untested, and the port in general is still work in progress. We have
+an upcoming blog post that will be going into more detail about the porting
+process, implementation details, and the obstacles we faced along the way.
+
+### Rust Support in User Space
+
+Project member [@64](https://github.com/64) has been working to bring Rust to Managarm. So far, this has consisted of:
+* Adding information about our `x86_64-unknown-managarm-system` target to [rustc](https://github.com/rust-lang/rust) (so that it can link with our LLVM and such). This initial work was done by [@avdgrinten](https://github.com/avdgrinten).
+* Adding bindings for [mlibc](https://github.com/managarm/mlibc) in Rust's [libc](https://github.com/rust-lang/libc) crate ([managarm/bootstrap-managarm#96](https://github.com/managarm/bootstrap-managarm/pull/96/files#diff-3d13f40b9e85b21978241de17ebb802857f04bf1af5b1f0d80f1c76f50d040d2)).
+* Patching Rust's standard library to support Managarm, and stubbing out functionality that we do not implement yet ([managarm/bootstrap-managarm#96](https://github.com/managarm/bootstrap-managarm/pull/96/files#diff-b086cf11aa261f991d7eb66b8164907ad9ee5bc4165cbc1739d65fcd2607be78)).
+* Integrating [cargo](https://github.com/rust-lang/cargo) with our build system so that we can cross-compile crates for Managarm ([managarm/xbstrap#48](https://github.com/managarm/xbstrap/pull/48)).
+* Porting [ripgrep](https://github.com/BurntSushi/ripgrep) and [exa](https://github.com/ogham/exa) ([managarm/bootstrap-managarm#102](https://github.com/managarm/bootstrap-managarm/pull/102)).
```

Comment:
```suggestion
* Porting [ripgrep](https://github.com/BurntSushi/ripgrep), [exa](https://github.com/ogham/exa) and [alacritty](https://github.com/alacritty/alacritty) ([managarm/bootstrap-managarm#102](https://github.com/managarm/bootstrap-managarm/pull/102), [managarm/bootstrap-managarm#180](https://github.com/managarm/bootstrap-managarm/pull/180)).
```

Reply:
As noted elsewhere, I'd rather not add alacritty and move this to the next one. That decision, however, isn't up to me.

Reply:
Well, it's a little odd to call it the August 2022 post if we're only talking about things that happened before January

Reply:
Agree. Hence why I wanted it out in January. However, we need to be weird somewhere or have an end of year post of 4 months of work. Also, if we want this to be accurate to current work, more content needs to be written, leading to delays, and I want it out ASAP.